src/Writer/Pgsql.php

```diff
@@ -21,7 +21,7 @@ class Pgsql extends Writer implements WriterInterface
 
     /** @var array $allowedTypes */
     private static $allowedTypes = [
-        'int', 'smallint', 'integer', 'bigint',
+        'int', 'smallint', 'integer', 'bigint', 'uuid',
```

Comment:
Ahoj. To pole sa nikde nepouziva, je to nejaky historicky pozostatok.

Je to task na UI, aby si mohol user vybrat typ zo zoznamu.
V idealnom pripade nic viac nebude potrebne menit (ak to `uuid` nefunguje nejak velmi specialne).

Reply:
aha ok, na upravu ui sa chystam ale vsimol som si podobne pole aj tu tak som to pridal. Tak tu to zavriem a upravim iba na ui, diky ;)